.github/workflows/sync-examples.yml

```diff
@@ -1,12 +1,8 @@
 name: '[CI] Sync Examples'
 
 on:
-    push: # Only run on merges into main that modify files under examples/
-        branches:
-        -   main
-        paths:
-        -   examples/**
-        -   .github/workflows/sync-examples.yml # Test that workflow works when changed
+    release:
+        types: [published]
```

Comment:
Hey @pww217, sorry, could you please describe what this is doing?
Just so I can understand this part of the code =)
I understand what's the high level idea of the PR, just don't know exactly how this line is working.
I looked for it on the GH docs but couldn't find this specific command. (Not sure if I never found the real docs or if their docs are just bad like that?)
Thank you!

Reply:
So basically when we publish a new release from a draft it will trigger this. It's the same trigger as the latest/versioned build artifacts, so they'll be in lockstep.

Reply:
If you're all set I can merge this one with approval.

Reply:
Aye aye 🏴‍☠️